README.md

```diff
@@ -1,47 +1,45 @@
+Black Duck Detect is an intelligent scan client that analyzes code in your projects and associated folders to perform compositional analysis. Detect can be configured to send scan results to Black Duck SCA, which generates risk analysis when identifying open-source components, licenses, and security vulnerabilities.
 
 ## Build
 
-[![Build Status](https://travis-ci.org/blackducksoftware/hub-gradle-plugin.svg?branch=master)](https://travis-ci.org/blackducksoftware/synopsys-detect)
-[![Coverage Status](https://coveralls.io/repos/github/blackducksoftware/synopsys-detect/badge.svg?branch=master)](https://coveralls.io/github/blackducksoftware/synopsys-detect?branch=master)
+<!--[![Build Status](https://travis-ci.org/blackducksoftware/hub-gradle-plugin.svg?branch=master)](https://travis-ci.org/blackducksoftware/synopsys-detect)-->
+<!--[![Coverage Status](https://coveralls.io/repos/github/blackducksoftware/synopsys-detect/badge.svg?branch=master)](https://coveralls.io/github/blackducksoftware/synopsys-detect?branch=master)-->
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Black Duck Security Risk](https://copilot.blackducksoftware.com/github/repos/blackducksoftware/synopsys-detect/branches/master/badge-risk.svg)](https://copilot.blackducksoftware.com/github/repos/blackducksoftware/synopsys-detect/branches/master)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.synopsys.integration%3Asynopsys-detect&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.synopsys.integration%3Asynopsys-detect)
+<!--[![Black Duck Security Risk](https://copilot.blackducksoftware.com/github/repos/blackducksoftware/synopsys-detect/branches/master/badge-risk.svg)](https://copilot.blackducksoftware.com/github/repos/blackducksoftware/synopsys-detect/branches/master)-->
+<!--[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.synopsys.integration%3Asynopsys-detect&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.synopsys.integration%3Asynopsys-detect)-->
 
 ## Where can I get the latest release?
 
-*Available from GitHub for Linux/MacOS by running:*
+*Available for Linux/MacOS by running:*
 
 ```bash
-bash <(curl -s -L https://detect.synopsys.com/detect9.sh)
+bash <(curl -s -L https://detect.blackduck.com/detect10.sh)
 ```
 
-*Available from GitHub for Windows by running in **command prompt**:*
+*Available for Windows by running in **command prompt**:*
 
 ```cmd
-powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.synopsys.com/detect9.ps1?$(Get-Random) | iex; detect"
+powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.blackduck.com/detect10.ps1?$(Get-Random) | iex; detect"
```

Comment:
All of these changes are good. But we probably can't merge this to gh-pages branch yet (there is a automatic deployment job as you know) as https://detect.blackduck.com/detect10.ps1 will only be available after 10.0.0 release

Reply:
My bad, didn't notice this is only in Draft. Ignore the above comment